### DISCUSSION TITLE
how to remove scrollto () in the element interface, because scrollto does not exist in the DOM element in some devices.

### DISCUSSION THREAD

**Discussion by cyesky** [vitest-dev/vitest#1955] [Q&A]
When executing unit test cases, how to remove the methods contained in a DOM to meet the requirements of compatibility testing. For example, how to remove scrollto () in the element interface, because scrollto does not exist in the DOM element in some devices.

Reference link:

https://caniuse.com/?search=scrollto

https://developer.mozilla.org/zh-CN/docs/Web/API/Element/scrollTo

**Comment by sheremet-va** [comment]
I don't think `scrollTo` exists in JSOM even. You can change any method, using prototype:

```ts
Element.prototype.scrollTo = null // whatever you want
```
